src/theory/quantifiers_engine.h

```diff
@@ -56,6 +56,8 @@ namespace quantifiers {
   class TermUtil;
   class FirstOrderModel;
   class QuantAttributes;
+  class QuantEPR;
+  class QuantRelevance;
```

Comment:
Include corresponding header files.

Reply:
Currently the convention I'm using is to forward declare modules that QuantifiersEngine owns, so that the modules can include quantifiers_engine.h if necessary, so the includes are intentionally omitted here to avoid cycles.  This may be revisited in the future.

Reply:
Is this even done in the modules? If this is the case you maybe have to move some code from the quantifiers engine to a separate util file?

Reply:
I'd have to check, it definitely was the case before.  Perhaps we can open a github issue to revisit this choice?

Reply:
Sounds good.

Reply:
Actually, the h files of several modules do include quantifiers_engine.h.
Regardless I've opened #1307 and referenced it.